File: tests/human/x/python/cross_join_filter.py
nums = [1, 2, 3]
letters = ["A", "B"]
pairs = [{"n": n, "l": l} for n in nums for l in letters if n % 2 == 0]
print("--- Even pairs ---")
for p in pairs:
    print(p["n"], p["l"])

File: tests/human/x/swift/cross_join_filter.swift
let nums = [1, 2, 3]
let letters = ["A", "B"]
var pairs: [(Int, String)] = []

for n in nums where n % 2 == 0 {
    for l in letters {
        pairs.append((n, l))
    }
}

print("--- Even pairs ---")
for p in pairs {
    print(p.0, p.1)
}
